pkgs/applications/networking/cluster/argocd-autopilot/default.nix

```diff
@@ -0,0 +1,52 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "argocd-autopilot";
+  version = "0.2.30";
+
+  src = fetchFromGitHub {
+    owner = "argoproj-labs";
+    repo = "argocd-autopilot";
+    rev = "v${version}";
+    sha256 = "sha256-UKVAitxhBXbVEwiEC9NTQNwZQlZAYUMpd8Po7XLf2UU=";
+  };
+
+  vendorSha256 = "sha256-i4fBfQugkXdynLxjXSEj9mErrJ0LeUJ/dipcV5HoWxg=";
```

Comment:
Hmm I think the correct sha is `sha256-0bWmmZ0eGLHRzkNU75qg7GS3XfsR7p84IBNluEtk8Es=` (local built). Other than that, everything seems OK.

- https://github.com/NixOS/nixpkgs/pull/147388/checks?check_run_id=5496113280

Reply:
That doesn't work for me on macOS:

<img width="871" alt="Képernyőfotó 2022-03-11 - 10 32 28" src="https://user-images.githubusercontent.com/1226384/157840607-bb4e4004-99eb-4770-9cbd-543b915c2886.png">

Reply:
I have no idea about macOS, but maybe is something along these lines?

https://github.com/golang/go/issues/29282

Could you clean nix cache (`nix-collect-garbage -d; nix-store --optimise;`) and enable sandbox for nix (se OP)?

Reply:
Did all these, enabled sandbox and I'm still getting a hash mismatch for the modules.